OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectDatum.cs

```diff
@@ -1,14 +1,31 @@
-﻿using Robust.Shared.IoC;
+﻿using OpenDreamRuntime.Procs;
+using OpenDreamShared.Dream;
+using Robust.Shared.IoC;
 
 namespace OpenDreamRuntime.Objects.MetaObjects {
     class DreamMetaObjectDatum : DreamMetaObjectRoot {
         public override bool ShouldCallNew => true;
+        private bool _inDatumList = false;
```

Comment:
There only exists a single meta-object for each type. This isn't correctly keeping track of what datums are in the datum list, which isn't necessary because `OnObjectCreated()` and `OnObjectDeleted()` should only be called once for each DreamObject.